docs/dev/connectors/kafka.md

```diff
@@ -466,12 +445,12 @@ Through the producer record you can:
 ### Kafka Producers and Fault Tolerance
 
 <div class="codetabs" markdown="1">
-<div data-lang="Universal and 011" markdown="1">
-With Flink's checkpointing enabled, the `FlinkKafkaProducer011` (`FlinkKafkaProducer` for Kafka >= 1.0.0 versions) can provide
+<div data-lang="Universal" markdown="1">
```

Comment:
How about we remove the Tabs whatsoever? IMO it does not make sense anymore for a single connector.

Reply:
I don't know how that happened but I thought I had already removed them. Good catch!